CHANGELOG.md

```diff
@@ -20,60 +20,79 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## [Unresolved]
 
  ### Security
- 
+
+ ### Added
+
+ ### Changed
+
+ ### Removed
+
+ ### Fixed
+
+## [0.1.3] - 2020-07-30
+
+ ### Security
+
  ### Added
-  
+ - [deploy-cloudrun]: Added action for deploying to Google Cloud Run
```

Comment:
It would be great if we can link PR#s here.

Reply:
Added